sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -3691,6 +3691,22 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
       checkAnswer(sql("SELECT id FROM t WHERE (SELECT true)"), Row(0L))
     }
   }
+
+  test("SPARK-33233: CUBE/ROLLUP/GROUPING SETS support GROUP BY ordinal") {
+    withTable("t") {
+      sql("CREATE TABLE t USING PARQUET AS SELECT id AS a, id AS b, id AS c FROM range(1)")
```

Comment:
Since `group-by-ordinal.sql` looks enough for end-2-end tests, I think we don't need to update this file.

Reply:
> Since `group-by-ordinal.sql` looks enough for end-2-end tests, I think we don't need to update this file.

Yea.